test/unit/TestNXOSDriver.py

```diff
@@ -71,10 +73,12 @@ def show(self, command, fmat = 'xml', text = False):
             extension = 'txt'
 
         filename = re.sub(r'[\[\]\*\^\+\s\|\/]', '_', command)
-        mock_file = 'nxos/mock_data/{filename}.{extension}'.format(
+        mock_file = '{filename}.{extension}'.format(
             filename  = filename[0:150],
             extension = extension
         )
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
This is fine too -- I do not have any personal preferences, just that using inspect adds an additional import used in one single place. 
I am now aware of the differences, would you mind pointing here briefly some advantages? I only know that ```__file__``` is not available in the CLI, but using inspect you have the standard input ```stdin``` as replacement. 
If it has some benefits, then we should leave it like you proposed.